mobile/src/components/course/components/module/module.tsx

```diff
@@ -9,19 +11,41 @@ type Props = {
   index: number;
   title: string;
   description: string | null;
+  isMentor: boolean;
+  moduleId: number;
+  tasks: TaskWithModuleResponseDto[];
 };
 
-const Module: FC<Props> = ({ index, title, description }): ReactElement => {
+const Module: FC<Props> = ({
+  index,
+  title,
+  description,
+  isMentor,
+  moduleId,
+  tasks,
+}): ReactElement => {
   const { width } = useWindowDimensions();
   const moduleSequenceNumber = `${index + 1}.`;
 
+  const moduleTask = tasks.find((task) => task.moduleId === moduleId);
+
   return (
     <View style={styles.container}>
       <View style={styles.indexWrapper}>
         <Text style={styles.index}>{moduleSequenceNumber}</Text>
       </View>
       <View style={styles.textWrapper}>
-        <Text style={styles.title}>{title}</Text>
+        <View style={styles.titleWrapper}>
+          <Text style={styles.title}>{title}</Text>
+          {isMentor && moduleTask && (
```

Comment:
Move this check outside
```suggestion
          {showStatus && (
```